client/app/styles/tip.css

```diff
@@ -23,3 +24,36 @@
 .fieldAnswer {
   margin-bottom: 10px;
 }
+
+.SubmissionInfoSection {
+  background-color: #eaebe2;
+  border-radius: 2%;
```

Comment:
in relation to borders never use percentage as its obviously a related to the size of the box so that in the end every box will have fancy different style